.env.example

```
@@ -53,8 +53,16 @@ MASTER_KEY=""
 
-# Start management profile services (agent, socket-proxy)
-COMPOSE_PROFILES="management"
+# Start management profile services (agent, socket-proxy) and Pocket ID for local OIDC dev.
+# Remove "oidc" if you don't need the local Pocket ID instance.
+COMPOSE_PROFILES="management,oidc"
+
+# Pocket ID local dev OIDC provider (oidc profile only).
+# The seeder uses STATIC_API_KEY to provision groups, users, and an OIDC client, then writes
+# .env.development.local (auto-loaded by bun dev) and data/dev-oidc-logins.txt (one-time login URLs).
+# These values are dev-only; rotate them in any non-local environment.
+POCKET_ID_API_KEY="dev-static-api-key-32chars!!!!!"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)